WordPressKit/ShareAppContentServiceRemote.swift

```diff
@@ -0,0 +1,38 @@
+/// Encapsulates logic for fetching content to be shared by the user.
+///
+open class ShareAppContentServiceRemote: ServiceRemoteWordPressComREST {
+    /// Fetch content to be shared by the user, based on the provided `appName`.
+    ///
+    /// - Parameters:
+    ///   - appName: A string that identifies the app to be shared.
+    ///   - completion: A closure that will be called when the fetch request completes.
+    open func getContent(for appName: String, completion: @escaping (Result<RemoteShareAppContent, Error>) -> Void) {
```

Comment:
Hey @dvdchr .

I think it would be nice to have an `enum` for the `appName` instead of just a `String`. Maybe something like:

```
public enum ShareAppName: String {
    case wordpress = "wordpress"
    case jetpack = "jetpack"
}
```

That way, it would be restricted to known options, and it would be clear what those options are. What do you think?

Reply:
I previously considered whether it's better to use a String enum, or to leave it stringly-typed to leave the context out of the kit repo – it was kind of a 50:50 for me. But I've no problem adjusting this to use enum instead. 🙂

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/433/commits/08deac22ecf1aee9219619c9839707a02a0a1c8a !